src/panels/TimePanel/SimulationIncrement.tsx

```diff
@@ -41,14 +48,16 @@ export function SimulationIncrement() {
     <>
       <Group grow mb={'xs'}>
         <Select
-          label={'Display Unit'}
+          label={t('simulation-increment.select-unit-label')}
           value={stepSize}
           data={selectableData}
           allowDeselect={false}
           onChange={(value) => setStepSize(value! as TimePart)}
         />
         <NumericInput
-          label={`${stepSize} / second`}
+          label={`${t(`time-parts.${stepSize}`, {
```

Comment:
This is a little magic - the fact that the key has to match enum string. It took me a while to actually find where the conversion was happening. 

How about making a function or something in util.ts that maps from the enum string to the correct i18next key? It should also throw if a provided key does not exist (important!)

And, as mentioned above, then make the keys lowercase. This would be a lot more explicit, and hence less magic and more understandable. As well as reduce the risk for errors in the case where we for example decide to change the keys for the enum or add another time unit in the future